index.html

```diff
@@ -28,23 +39,40 @@
     <main>
       <section class="section1">
         <header>
-          <nav class="toolbar">
-            <h2 class="logo"><a href="#">My Logo</a></h2>
-            <a href="#"
-              ><img class="menu-icon" src="images/Union.svg" alt="menu-icon"
-            /></a>
+          <nav>
+            <div class="toolbar">
+              <h2 class="logo"><a href="#">My Logo</a></h2>
+              <ion-icon class="open" name="menu-outline"></ion-icon>
+            </div>
+            <div class="menu-links">
+              <ul class="menu-ul">
+                <li class="ctc-link"><a href="#portfolio">Portfolio</a></li>
+                <li class="ctc-link"><a href="#about">About</a></li>
+                <li class="ctc-link ctc-last">
+                  <a href="#contact">Contact</a>
+                </li>
+              </ul>
+              <ion-icon class="close" name="close-outline"></ion-icon>
+              <img
+                class="bar"
+                src="/images/Shape.png"
+                alt="horizontal-bar"
+              />
+            </div>
```

Comment:
Kindly make sure that your design is as close as possible to Figma's design. There is an issue with the height of your `menu-links` class. As you can see, it does not cover the entire mobile screen. Sadly, your image for the horizontal bar is not loading 😥check its path 😉 Also you can try and make your opacity a little bit stronger. Please try with the following value so that the text behind the menu is barely seen (0.96) 👀


### Your Design 
![Captura](https://user-images.githubusercontent.com/15807343/151021802-b5543b63-d380-4970-8ecd-a9bf7e1b64c1.PNG)

### Figma's
![image](https://user-images.githubusercontent.com/15807343/151021755-b6659cc7-3236-48a8-9dcb-767f2560f13f.png)